apps/files/src/components/Collaborators/Collaborator.vue

```diff
@@ -69,7 +69,7 @@
         </oc-button>
         <oc-spinner
           v-else-if="$_loadingSpinnerVisible"
-          :aria-label="$gettext('Removing collaborator…')"
+          :aria-label="$gettext('Removing person…')"
```

Comment:
When adding new people, the spinner has label without those three dots. Can we remove them here as well for consistency?

Reply:
done